test/python/expected/pyodbc/ddl_all_objects.out

```diff
@@ -151,8 +151,8 @@ ALTER TABLE [dbo].[test_tsql_const] ADD  CONSTRAINT [test_tsql_const_pkey] PRIMA
 )WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, SORT_IN_TEMPDB = OFF, IGNORE_DUP_KEY = OFF, ONLINE = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON)
 GO
 
-SET ANSI_NULLS ON
-SET QUOTED_IDENTIFIER ON
+SET ANSI_NULLS OFF
+SET QUOTED_IDENTIFIER OFF
```

Comment:
Why are we changing this?

Reply:
As mentioned in the description, Updated DDL export tests output where ANSI_NULLS and QUOTED_IDENTIFIER are set to OFF in DDL scripting of triggers. It doesn't affect the scripting, we can take this up later.